app/javascript/dashboard/components/layout/Sidebar.vue

```diff
@@ -55,11 +59,12 @@
           </ul>
         </div>
       </transition>
+
       <div class="current-user" @click.prevent="showOptions()">
         <thumbnail
           :src="currentUser.avatar_url"
           :username="currentUserAvailableName"
-          :status="currentUser.availability_status"
+          :status="null"
```

Comment:
Setting this not null doesn't make sense. I think we can remove this key altogether.

Reply:
Removed